### PR TITLE
Update butterknife

### PR DESCRIPTION
**Pull request by jaredsburrows**
@JakeWharton 
Updated butterknife to 8

### CODE REVIEW

File: telecine/build.gradle [hunk 1]
```diff
@@ -72,10 +72,10 @@ android {
 }
 
 dependencies {
-  compile 'com.android.support:support-annotations:23.2.1'
+  compile 'com.android.support:support-annotations:23.4.0'
   compile 'com.android.support:design:23.4.0'
-  compile 'com.google.android.gms:play-services-analytics:8.4.0'
-  compile 'com.jakewharton:butterknife:7.0.1'
+  compile 'com.google.android.gms:play-services-analytics:9.0.1'
```
Comment (JakeWharton) [inline telecine/build.gradle]: Revert

**Comment by jaredsburrows** [comment]
@JakeWharton Fixed.

File: telecine/build.gradle [hunk 2]
```diff
@@ -75,7 +75,7 @@ dependencies {
   compile 'com.android.support:support-annotations:23.2.1'
   compile 'com.android.support:design:23.4.0'
   compile 'com.google.android.gms:play-services-analytics:8.4.0'
-  compile 'com.jakewharton:butterknife:7.0.1'
+  compile 'com.jakewharton:butterknife:8.0.1'
```
Comment (JakeWharton) [inline telecine/build.gradle]: Needs the `apt` dependency on 'butterknife-compiler' or will fail at runtime.

Comment (jaredsburrows) [inline telecine/build.gradle]: Ah, good eye.

**Pull request merged**